irods_specific_implementation/CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ foreach(API_NAME ${API_SERVER_NAMES})
         ${IRODS_EXTERNALS_FULLPATH_BOOST}/lib/libboost_system.so
     )
 
-    target_compile_definitions(${EXECUTABLE_NAME} PRIVATE SPDLOG_FMT_EXTERNAL SPDLOG_NO_TLS)
+    target_compile_definitions(${EXECUTABLE_NAME} PRIVATE SPDLOG_FMT_EXTERNAL) # TODO SPDLOG_NO_TLS)
```

Comment:
Think about whether the `SPDLOG_NO_TLS` is needed.

Reply:
thread local storage

https://github.com/gabime/spdlog/blob/8826011c81b69e1f8427be520b357c19c74ea480/include/spdlog/tweakme.h#L32-L39

Reply:
Yes. I commented it out because of that. I doubt this project is doing any forking. For that reason, this option should be safe to remove.